app/auth-portal/src/pages/WorkspaceAuthTokensPage.vue

```diff
@@ -124,14 +132,19 @@ import {
   VButton,
   LoadStatus,
   ErrorMessage,
+  Modal,
+  IconButton,
 } from "@si/vue-lib/design-system";
 import { useAsyncState } from "@vueuse/core";
 import { apiData } from "@si/vue-lib/pinia";
+import { useHead } from "@vueuse/head";
 import { useWorkspacesStore, WorkspaceId } from "@/store/workspaces.store";
 import WorkspacePageHeader from "@/components/WorkspacePageHeader.vue";
 import { useAuthTokensApi } from "@/store/authTokens.store";
 import AuthTokenListItem from "@/components/AuthTokenListItem.vue";
 
+useHead({ title: "Automation Tokens" });
```

Comment:
Should I be doing this on all pages?

Reply:
@jkeiser This can be done on the router too, but on the auth portal we've been using this pattern. I'd say useHead on the auth portal, the router title on the webapp